doc/classes/DisplayServer.xml

```diff
@@ -995,6 +978,19 @@
 			<description>
 			</description>
 		</method>
+		<method name="window_set_vsync_mode">
+			<return type="void">
+			</return>
+			<argument index="0" name="vsync_mode" type="int" enum="DisplayServer.VSyncMode">
+			</argument>
+			<argument index="1" name="window_id" type="int" default="0">
+			</argument>
+			<description>
+				Sets the VSync mode of the given window.
+				See [enum DisplayServer.VSyncMode] for possible values and how they affect the behavior of your application.
+				Depending on the platform and used renderer, the engine will fall back to [code]Enabled[/code], if the desired mode is not supported.
```

Comment:
I know there was some back and forth on this but it seems @clayjohn wasn't familiar with this convention: you can refer to constants directly with `[constant VSYNC_ENABLED]` to create a hyperlink.

Reply:
Changed.